src/absolute.ts

```diff
@@ -0,0 +1,11 @@
+import type { Transformer } from 'integreat'
+import { parseNum } from './utils/cast.js'
+import { isNumber } from './utils/is.js'
+
+const transformer: Transformer = () => () => (value) => {
+  const parsedValue = parseNum(value)
+  return isNumber(parsedValue) && !Number.isNaN(parsedValue)
```

Comment:
Do you get NaN this from `parseNum()`? In that case, I think you should fix that in `parseNum` instead, because that is not intended.

Reply:
From conversation outside the PR: There is another number parser function in this package that also coerces other type of values to numbers. I though we were talking about that one, but this is fine as it is now.